Jenkinsfile.release

```
@@ -322,22 +300,21 @@ pipeline {
                     }
                     addStringParam(buildParams, 'EXAMPLES_URI', env.EXAMPLES_URI)
                     addStringParam(buildParams, 'EXAMPLES_REF', env.EXAMPLES_REF)
-                    addStringParam(buildParams, 'MAVEN_ARTIFACT_REPOSITORY', env.STAGING_REPOSITORY) // Get artifacts from staging repository
+                    addStringParam(buildParams, 'MAVEN_ARTIFACT_REPOSITORY', getArtifactsRepositoryParam() ?: env.DEFAULT_STAGING_REPOSITORY) // Get artifacts from staging repository
 
                     buildJob(OPERATOR_DEPLOY, buildParams)
                 }
             }
         }
 
-        stage('Is staging repository released?') {
+        stage('Are staged artifacts released?') {
             when {
-                expression { return !isCreateReleaseBranchesOnly() && !getArtifactsRepositoryParam() } // Execute only if artifacts repository was not given, which means the staging repository has been created
+                expression { return !isCreateReleaseBranchesOnly() && isArtifactsDeploy() && !getArtifactsRepositoryParam() } // Execute only if artifacts repository was not given, which means the staging repository has been created
```

Just nitpicking to make the comment fit the screen.
```suggestion
                // Execute only if artifacts repository was not given, which means the staging repository has been created.
                expression { return !isCreateReleaseBranchesOnly() && isArtifactsDeploy() && !getArtifactsRepositoryParam() } 
```